crates/bevy_reflect/bevy_reflect_derive/src/enum_utility.rs

```diff
@@ -1,6 +1,7 @@
 use crate::derive_data::{EnumVariantFields, ReflectEnum};
-use proc_macro2::Ident;
+use proc_macro2::{Ident, TokenStream};
```

Comment:
For reference:
There are two crates: `proc_macro` and `proc_macro2`, each exporting the `TokenStream` struct. It seems, the bevy code base is very careful to only import `TokenStream` from `proc_macro` and not `proc_macro2`, in fact, when it does import `proc_macro2::TokenStream` it aliases it with `proc_macro2::TokenStream as TokenStream2`. I've no idea why it's done this way. I think it's because `TokenStream` might be used bare in `quote!` macros, and therefore, if we overwrite it with the one from `proc_macro2`, it will lead to bad bugs.

Reply:
I’m not sure it leads to any bugs tbh (`quote` returns `proc_macro2` so my guess is it uses that internally as well). 

I think it's because Rust expects the `proc_macro` version as the return type to macro functions. And if other Bevy devs are like me, and copy+paste macro functions from online (😅), then those typically just put a plain `TokenStream` as the return type rather than `proc_macro::TokenStream`.